[DOCS-12495] Update Set Up Pipelines (#33060)
DataDog/documentation: content/en/observability_pipelines/configuration/set_up_pipelines.md

    - [Sensitive Data Redaction][9]
    - [Split Logs][10]
1. Select and set up your [source][1].
1. Add [processors][2] to transform, redact, and enrich log data.
    - If you want to copy a processor, click the copy icon for that processor and then use `command-v` to paste it.
1. Select and set up [destinations][3] for your processed logs.

### Add or remove components

#### Add another processor group

{{< img src="observability_pipelines/setup/another_processor_group.png" alt="The Pipelines page showing two processor groups sending logs to the same destination" style="width:100%;" >}}

If you want to add another group of processors for a destination:
1. Click the plus sign (**+**) at the bottom of the existing processor group.
1. Click the name of the processor group to update it.
1. Optionally, enter a group filter. See [Filter Syntax][17] for more information.
1. Click **Add** to add processors to the group.
1. If you want to copy all processors in a group and paste them into the same processor group or a different group:
    1. Click the three dots on the processor group.
    1. Select **Copy all processors**.

>     1. Select the processor group you want to paste the processors into and then use `command-v` to paste them.

Reviewer: ```suggestion
    1. Select the desired processor group, and then paste the processors into it.
```

Reviewer: Suggesting this because `command-v` is not an option for Windows users. I recognize this is not part of your update, and I'm open to your feedback if you have a better way to communicate/phrase this.

1. You can toggle the switch to enable and disable the processor group and also each individual processor.

> <br>**Notes**:

Reviewer: With the preceding blank line, this line break doesn't seem necessary. In the [preview](https://docs-staging.datadoghq.com/may/update-set-up-pipelines/observability_pipelines/configuration/set_up_pipelines/?tab=pipelineui#add-another-processor-group), this creates two blank lines, which is not consistent with the other **[Notes](https://docs-staging.datadoghq.com/may/update-set-up-pipelines/observability_pipelines/configuration/set_up_pipelines/?tab=pipelineui#add-another-destination-to-a-processor-group)** listed in this section. Feel free to disregard this suggestion if that's intentional.
